packages/python/src/components/Value.tsx

```diff
@@ -0,0 +1,62 @@
+import { For, Indent, memo } from "@alloy-js/core";
+
+export interface ValueProps {
+  value?: unknown;
+}
+
+export function Value(props: ValueProps): any {
```

Comment:
We typically want to stick to the python names for things, this appears to be an Atom, consider naming it that?